flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/AccessExecutionGraph.java

```diff
@@ -47,6 +47,13 @@ public interface AccessExecutionGraph extends JobStatusProvider {
      */
     String getJsonPlan();
 
+    /**
+     * Returns the stream graph as a JSON string.
+     *
+     * @return stream graph as a JSON string, or empty string if the job submitted with JobGraph.
```

Comment:
or an empty string if the job is submitted with a JobGraph or if it's a streaming job.

Reply:
Updated.